kernelci.org/content/en/architecture/_index.md

```diff
@@ -4,24 +4,24 @@ date: 2024-07-03
 description: "Learn the inner details behind the KernelCI systems"
 ---
 
-![architecture](/image/kernelci-architecture.png)
+![architecture](kernelci-architecture.svg)
 
 The first thing worth noting here is that there are two main parts of the
 overall KernelCI architecture:
 
-### [API & Pipeline](../api_pipeline)
+### [Maestro](../maestro)
 
-The top half of this picture shows native services interacting directly with
-the API: LAVA test labs, Kubernetes clusters, custom test environments and the
-job scheduler.  These are referred to as the KernelCI Pipeline in a loose
-sense.  Such services can be run pretty much anywhere, they are just API
-clients with a particular purpose.
+The top left box of this picture shows Maestro. While Maestro is only one software system, architecturally its flow has 2 parts: (1) triggering builds and list of tests to run on each platform and (2) driving such tests.
+
+Maestro has a [pipeline-based](../maestro/pipeline) design and an [API](../maestro/api/) that allow other systems and tools to interact with it to subscribe to events (such as new build/test triggers) or send patches to be tested by the KernelCI infrastructure.
```

Comment:
>[pipeline-based](../maestro/pipeline)

This directory doesn't have an index file so nothing gets shown when you click on this link.

Reply:
Fixed.